docs/notes/pr-141-comment-burn-down.md

```
@@ -0,0 +1,100 @@
+<!-- SPDX-License-Identifier: Apache-2.0 OR MIND-UCAL-1.0 -->
+<!-- © James Ross Ω FLYING•ROBOTS <https://github.com/flyingrobots> -->
+# PR #141 — Comment Burn-Down
+
+PR: https://github.com/flyingrobots/echo/pull/141
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)